src/app/components/data-viewer/data-viewer.component.ts

```diff
@@ -315,7 +315,7 @@ export class DataViewerComponent implements OnInit, OnChanges {
           keys = [this.pageData.item.key];
         }
         this.redisService.call(this.pageData.id, [['DEL'].concat(keys)]).subscribe(() => {
-          this.util.showMessage('delete successful');
+          this.util.showMessage('delete successfully');
```

Comment:
may this please be changed to 'deleted successfully'?
Thanks :)

Reply:
Done!